src/backend/drm/DRM.cpp

```diff
@@ -1427,11 +1427,10 @@ bool Aquamarine::CDRMOutput::commitState(bool onlyTest) {
             backend->backend->log(AQ_LOG_ERROR, "drm: No mode on enable commit");
             return false;
         }
-    }
-
-    if (STATE.drmFormat == DRM_FORMAT_INVALID) {
-        backend->backend->log(AQ_LOG_ERROR, "drm: No format for output");
-        return false;
+        if (STATE.drmFormat == DRM_FORMAT_INVALID) {
```

Comment:
now it wont be checked if we commit just format :P

I'd do it below the format commit

```cpp
if (STATE.enabled) {
   // check
}
```

Reply:
yeah im gonna uninstall linux now ;-;

Reply:
lol